### DISCUSSION TITLE
1923

### DISCUSSION THREAD

**Discussion by masterpl4n** [nzp-team/nzportable#517] [Map Releases]
### 1923
A small stereotypical house map set in 1923. My first proper NZP map!
PLEASE NOTIFY ME OF ANY BUGS!

Story:
There are zombies

FEATURES:
3 Perks
Claustrophobic Layout
Bugs! (probably)
Time-period accurate weaponry
A cool shed barrier thing I'm proud of for some reason

TESTED ON:
PC, PSP

Thanks for playing!

Screenshots:
![image](https://github.com/nzp-team/nzportable/assets/143304484/7fe742ed-31ae-4620-b8e5-e966bbcacbfc)

![image](https://github.com/nzp-team/nzportable/assets/143304484/31b2308e-cbca-4413-963c-5819c9f5cfd2)

![image](https://github.com/nzp-team/nzportable/assets/143304484/9c8193e8-81d1-4d07-95fb-b20a89528398)

![image](https://github.com/nzp-team/nzportable/assets/143304484/690632bd-f9bb-41df-8d6c-7ae5e0d53839)

**Comment by Paindeluxe** [comment]
Not a bug but a suggestion

I would recommend you to add a light so that the map isn’t in fullbright mode

A warm golden glow in the fireplace would make for a nice ambiance

**Reply by DerpedCrusader** [reply]
Actually, he has some lights, but the map isn't enclosed

All he has to do is seal up the outside areas

And edit the light entities that are placed to have a brightness value

**Comment by MotoLegacy** [comment]
Hi, thanks for the release! In order to quality for meeting the installation standard, you'll have to restructure your `.ZIP` file. You can read more about that here: https://github.com/nzp-team/nzportable/discussions/159, (this is optional).

**Reply by masterpl4n** [reply]
Formatting is fixed now, thanks

**Comment by MotoLegacy** [comment]
As an aside, consider clipping your barricades!


https://github.com/nzp-team/nzportable/assets/26030250/27959468-d6ef-46b5-9f12-b4bfa18c6bf6

**Reply by masterpl4n** [reply]
How do I clip barricades? I'll reformat for the installation standard in a bit, thanks.

**Reply by MotoLegacy** [reply]
Place a brush in the hole in the wall and assign it the `CLIP` texture. Will fix the issue

**Comment by MotoLegacy** [comment]
Heads up! This does not meet install standard as the download is not hosted on archive.org
